src/components/PostPageDetail/index.js

```diff
@@ -554,7 +554,7 @@ const PostPageDetailIdComponent = (props) => {
   };
 
   const handleRefreshComment = async () => {
-    await getComment(false, true);
+    await getComment(false, false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the `getComment` function parameters from `(false, true)` to `(false, false)` may affect the functionality of the `handleRefreshComment` function. Please ensure that this change is intended and does not break any existing functionality.
```diff
-    await getComment(false, true);
+    await getComment(false, false);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->